src/py21cmfast/src/Stochasticity.c

```diff
@@ -789,7 +789,7 @@ int sample_halo_grids(gsl_rng **rng_arr, double redshift, float *dens_field,
     double total_volume_excluded = 0.;
     double total_volume_dexm = 0.;
     double cell_volume = VOLUME / pow((double)simulation_options_global->HII_DIM, 3);
-
+    bool OUT_OF_BUFFER = false;
```

Comment:
I would prefer lower-case for non defined/macro variables. I know this convention is not strictly kept throughout the code but let's try to move in that direction